### ISSUE TITLE
Add API to enable RabbitMQ mocking support

### ISSUE THREAD

**Issue by lbroudoux** [microcks/microcks-testcontainers-java#54]
### Reason/Context

Today, the Microcks `testcontainers`module only supports WebSocket, Kafka and AWS SQS and SNS as asynchronous protocols. The Microcks regular distribution supports all the other protocols like MQTT, Nats, Google PubSub and RabbitMQ.

The goal of this enhancement is to add RabbitMQ support to the `testcontainers` module. MQTT is a popular protocol and we met many developers being interested in having its support in Microcks testcontainers module.

### Description

Simply put, we need a new `withAMQPConnection(GenericConnection connection)` method on the `MicrocksAsyncMinionContainer` and the `MicrocksContainersEnsemble`.

**Comment by lbroudoux** [comment]
This is linked to microcks/microcks#1240
